WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -307,6 +310,9 @@ private void replaceIframeTag(final String tag, final String src) {
      * returns the full content, including CSS, that will be shown in the WebView for this post
      */
     private String formatPostContentForWebView(final String content, boolean renderAsTiledGallery) {
```

Comment:
Aha, thanks for the info @nbradbury .

I can be wrong but it seems that the post format can be user configurable and I wouldn't exactly feel "safe" to use it if so.

The fix here (and the parent PR) is directly related to tiled-galleries and is only in effect when certain CSS classes exist so I think the easiest discriminator is the CSS class name itself. I will add a quick lookup for the primary class name (`.tiled-gallery`) and we can change later if the flag you mention proves to be a better choice. Thanks!